docker-compose.yml

```diff
@@ -1,12 +1,13 @@
 services:
   database:
     image: ankane/pgvector
-    ports:
```

Comment:
Let's keep the `ports` fields in all the subservices. It makes it easier to manage if/when the ports need to be updated.

Reply:
@sabaimran 

I thought that exposing host port is easy to collide with other service of user. especially `5432`.
Is it better to change host port? like 42000 <= x <= 42999.

Reply:
Actually you're right, this should be fine. There's no *inherent* reason why we need the services exposed on the host machine, though it's helpful for debugging.